### PR TITLE
USWDS-Site - Select: Add changelog for select uswds#5268

### PR DESCRIPTION
**Pull request by mahoneycm**
# Summary

**Add changelog to select component page.** Summarizes https://github.com/uswds/uswds/pull/5268

## Related issue

USWDS PR: https://github.com/uswds/uswds/pull/5268

Closes uswds/uswds#5047

## Preview link

Select component page

### CODE REVIEW

**Comment by mahoneycm** [comment]
Closing in favor of #2096

**Review by mejiaj — CHANGES_REQUESTED** [review]
@mahoneycm can you fix conflicts and add reviewers when ready?

**Review by amyleadem — CHANGES_REQUESTED** [review]
@mahoneycm The changelog content looks good here. I'll approve once the build errors and merge conflicts resolve.

**Comment by mahoneycm** [comment]
@amyleadem @mejiaj Updated and should be ready to merge! The build failure is from the nonexistent `3.6.0` release link

**Comment by mahoneycm** [comment]
> **Warning**
> Changelog date must be updated

**Review by amyleadem — APPROVED** [review]
This looks good to me. 

Not particular to this particular PR, but I am wondering if we should start tagging `versionUswds` as N.N.N since it might not be as clear which release a change will belong to. No action needed at this time, just thinking out loud.

File: _data/changelogs/component-select.yml [hunk 1]
```diff
@@ -2,6 +2,18 @@ title: Select
...
+  summaryAdditional: This provides a clear indication to users that there is text being cut off by the select width.
+  isBreaking: false
+  affectsAccessibility: false
+  affectsGuidance: false
+  affectsJavascript: false
+  affectsMarkup: false
+  affectsStyles: true
+  githubPr: 5268
+  githubRepo: uswds
+  versionUswds: 3.6.0
```
Comment (thisisdano) [inline _data/changelogs/component-select.yml:16]:
```suggestion
  versionUswds: N.N.N
```

**Review by thisisdano — APPROVED** [review]
Updated base to release branch and used a dummy version number

**Pull request merged**